src/AppBundle/EventSubscriber/PullRequestSubscriber.php

```diff
@@ -19,18 +19,21 @@ public function setContainer(ContainerInterface $container)
     public static function getSubscribedEvents()
     {
         return [
-           'pullrequestevent_opened' => [
+            'pullrequestevent_opened' => [
                ['checkForTableDescription', 255],
                ['welcomePeople', 253],
                ['checkForNewTranslations', 252],
                ['initLabels', 254],
                ['checkCommits', 252],
-           ],
-           'pullrequestevent_edited' => [
+            ],
+            'pullrequestevent_edited' => [
                ['removePullRequestValidationComment', 255],
                ['removeCommitValidationComment', 255],
                ['checkForNewTranslations', 252],
             ],
+            'pullrequestevent_synchronize' => [
+                ['removeCommitValidationComment', 255],
```

Comment:
this fixes the non deletion of commit validation message when the pull request is synchronized.